kong/tools/request_aware_table.lua

```diff
@@ -5,37 +5,28 @@ local table_new = require("table.new")
 local table_clear = require("table.clear")
 local get_request_id = require("kong.tracing.request_id").get
 
+
 local is_not_debug_mode = (kong.configuration.log_level ~= "debug")
 
 
 local error        = error
 local rawset       = rawset
 local setmetatable = setmetatable
-local get_phase    = ngx.get_phase
-
-
-local NGX_VAR_PHASES = {
-  set           = true,
-  rewrite       = true,
-  access        = true,
-  content       = true,
-  header_filter = true,
-  body_filter   = true,
-  log           = true,
-  balancer      = true,
-}
```

Comment:
It feels like a better way is to implement a new `is_ngx_var_accessible` function and modify the `request_aware_table` and `request_id` to use that util function

Reply:
That is the original PR did, and I have discussed this point with sam, please refer to the comments above.

Reply:
@samugi , could you explain it here, thanks.

Reply:
I think this comment from @windmgc is outdated, these lines were removed, we don't need to check for ngx.var to be accessible anymore since we rely on simply getting the request id via the request_id.lua module now.